services/ymax-planner/src/gas-estimation.ts

```diff
@@ -0,0 +1,74 @@
+import type { AxelarChain } from '@agoric/portfolio-api/src/constants';
+import { gasLimitEstimates } from './support.ts';
+
+const AGORIC_CHAIN = 'agoric';
+const BLD_TOKEN = 'ubld';
+
+export type GasEstimator = ReturnType<typeof makeGasEstimator>;
+
+export const makeGasEstimator = ({
+  axelarApiAddress,
+  fetchFunc,
+  axelarChainIdMap,
+}: {
+  axelarApiAddress: string;
+  fetchFunc: typeof fetch;
+  axelarChainIdMap: Record<AxelarChain, string>;
+}) => {
+  const axelarEstimateGasAddress = `${axelarApiAddress}gmp/estimateGasFee`;
```

Comment:
Let's add some minor validation.
```suggestion
  axelarApiAddress.endsWith('/') ||
    Fail`Axelar API address must end with a slash: ${axelarApiAddress}`;
  try {
    new URL(axelarApiAddress);
  } catch (_err) {
    Fail`Invalid Axelar API address: ${axelarApiAddress}`;
  }
  const axelarEstimateGasAddress = `${axelarApiAddress}gmp/estimateGasFee`;
```